### PR TITLE
ヽ༼ຈل͜ຈ༽ﾉ

### PR DESCRIPTION
**Pull request by celsodantas**

### CODE REVIEW

File: game.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ const { readFileSync } = require('fs');
 const { Readme } = require('./lib/readme');
 
 const username = process.env["USER"]
+var delayedFunc = function () {}
```
Comment (epixa) [inline game.js]: Can you break these brackets onto separate lines?  See the legal/illegal moves thing in the readme

**Comment by kurtfunai** [comment]
pr name is gold

File: game.js (hunk 1 above)
Comment (patvice) [reply game.js]: @celsodantas 👍

**Comment by celsodantas** [comment]
sorry for the delay. done =D

File: game.js [hunk 2]
```diff
@@ -4,6 +4,9 @@ const { readFileSync } = require('fs');
 const { Readme } = require('./lib/readme');
 
 const username = process.env["USER"]
+var delayedFunc = function () {
```
Comment (epixa) [inline game.js:7]: `var`. I hate you.

**Pull request merged**